apps/ai_reports/tasks.py

```diff
@@ -0,0 +1,57 @@
+import ast
+import logging
+
+import backoff
+import httpx
+from celery import shared_task
+from django.conf import settings
+
+from adhocracy4.comments.models import Comment
+from apps.ai_reports.models import AiReport
+
+client = httpx.Client()
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def get_classification_for_comment(comment_pk: int) -> None:
+    try:
+        comment = Comment.objects.get(pk=comment_pk)
+        response = call_ai_api(comment=comment)
+        if response.status_code == 200:
+            extract_and_save_ai_classifications(comment=comment, report=response.json())
+    except httpx.HTTPError as e:
+        logger.error("Error connecting to %s: %s", settings.AI_API_URL, str(e))
+
+
+def skip_retry(e: Exception) -> bool:
+    if isinstance(e, httpx.HTTPStatusError):
+        return 400 <= e.response.status_code < 500
+    return False
+
+
+@backoff.on_exception(
+    backoff.expo, httpx.HTTPError, max_tries=4, factor=2, giveup=skip_retry
+)
+def call_ai_api(comment: Comment) -> httpx.Response:
```

Comment:
since you only need the comment field of the Comment obj here, I wouldn't pass the whole object